src/sidebar/store/modules/groups.ts

```diff
@@ -157,16 +160,22 @@ function loadGroups(groups: Group[]) {
  */
 function startLoadingFocusedGroupMembers() {
   return makeAction(reducers, 'LOAD_FOCUSED_GROUP_MEMBERS', {
-    focusedGroupMembers: 'loading',
+    focusedGroupMembers: { status: 'loading' },
   });
 }
 
 /**
  * Update members for focused group.
  */
-function loadFocusedGroupMembers(focusedGroupMembers: GroupMember[] | null) {
+function loadFocusedGroupMembers(members: GroupMember[] | null) {
   return makeAction(reducers, 'LOAD_FOCUSED_GROUP_MEMBERS', {
-    focusedGroupMembers,
+    focusedGroupMembers:
+      members === null
+        ? { status: 'not-loaded' }
+        : {
+            status: 'loaded',
+            members,
+          },
```

Comment:
There should probably be two different actions, `loadFocusedGroupMembers(members: GroupMember[])` and `resetFocusedGroupMembers()`, but I'm leaving this one for backwards compatibility.